### PR TITLE
chore(install): include verbose and trace flags for loglevel

### PR DESCRIPTION
**Pull request by zlesnr**

### CODE REVIEW

File: internal/install/ux/spinner.go [hunk 1]
```diff
@@ -42,7 +42,7 @@ func (s *Spinner) Start(msg string) {
 
 func (s *Spinner) Stop() {
 	// Only stop the spinner of the log level is info or below.
-	if log.GetLevel() <= log.InfoLevel {
+	if log.IsLevelEnabled(log.InfoLevel) {
```
Comment (ctrombley) [inline internal/install/ux/spinner.go:45]: 👍

File: internal/install/ux/spinner.go [hunk 2]
```diff
@@ -31,7 +31,7 @@ func NewSpinner() *Spinner {
 
 func (s *Spinner) Start(msg string) {
 	// Only start the spinner of the log level is info or below.
-	if log.GetLevel() > log.InfoLevel {
+	if log.IsLevelEnabled(log.DebugLevel) {
```
Comment (ctrombley) [inline internal/install/ux/spinner.go:34]: 👍

File: internal/install/command.go [hunk 3]
```diff
@@ -73,4 +81,6 @@ func init() {
 	Command.Flags().BoolVarP(&skipIntegrations, "skipIntegrations", "r", false, "skips installation of recommended New Relic integrations")
 	Command.Flags().BoolVarP(&skipLoggingInstall, "skipLoggingInstall", "l", false, "skips installation of New Relic Logging")
 	Command.Flags().BoolVarP(&testMode, "testMode", "t", false, "fakes operations for UX testing")
+	Command.Flags().BoolVar(&verbose, "verbose", false, "increase log level verbosity")
```
Comment (ctrombley) [inline internal/install/command.go:84]: Consider long flag `debug`, "debug level logging" for description and short flag `v`?

File: internal/install/command.go [hunk 4]
```diff
@@ -73,4 +81,6 @@ func init() {
...
+	Command.Flags().BoolVar(&trace, "trace", false, "trace level logging")
```
Comment (ctrombley) [inline internal/install/command.go:85]: Consider short flag `vv`?

File: internal/install/command.go (hunk 3 above)
Comment (zlesnr) [reply internal/install/command.go:84]: `-v` is taken for verbose currently on the root command, which we could change, but `--debug` sounds find to me too.

File: internal/install/command.go (hunk 4 above)
Comment (zlesnr) [reply internal/install/command.go:85]: I did and then read https://github.com/spf13/cobra/issues/679

**Review by ctrombley — APPROVED** [review]

**Pull request merged**